From: Peter Michael Green <plugwash@raspbian.org>
Subject: Automatically generated patch (5.9.1+dfsg-9+rpi1)

Last (up to) 3 git changes, FYI:

commit 9305cb3d174b9347b18735f7fbb6412e1e99c75c
Author: Peter Michael Green <plugwash@raspbian.org>
Date:   Sun Sep 3 13:02:16 2017 +0000

    Remove broken patch from quilt series.

commit 4f406fb7400e0cfdf6c88c1fa01534f063a961c0
Merge: 8bf70a2c 802d6f6e
Author: Peter Michael Green <plugwash@raspbian.org>
Date:   Sun Sep 3 12:56:50 2017 +0000

    Manual merge of version 5.7.1+dfsg-4+rpi1 and 5.9.1+dfsg-9 to produce 5.9.1+dfsg-9+rpi1

commit 802d6f6edac7e3257bac155727bf55623b7cee34
Merge: dd9ca752 41b462b6
Author: Dmitry Shachnev <mitya57@debian.org>
Date:   Sun Aug 20 17:13:27 2017 +0100

    Merge qtbase-opensource-src (5.9.1+dfsg-9) import into refs/heads/workingbranch
---

--- qtbase-opensource-src-5.9.1+dfsg.orig/configure.json
+++ qtbase-opensource-src-5.9.1+dfsg/configure.json
@@ -894,14 +894,6 @@
                 { "type": "define", "name": "QT_COMPILER_SUPPORTS_MIPS_DSPR2", "value": 1 }
             ]
         },
-        "neon": {
-            "label": "NEON",
-            "condition": "(arch.arm || arch.arm64) && tests.neon",
-            "output": [
-                "privateConfig",
-                { "type": "define", "name": "QT_COMPILER_SUPPORTS_NEON", "value": 1 }
-            ]
-        },
         "mremap": {
             "label": "mremap()",
             "condition": "tests.mremap",
